gdp/column/column.py

```diff
@@ -1185,22 +1610,24 @@ def _build_feed_tray_energy_balance(m):
 
     @m.Constraint()
     def feed_tray_energy_balance(_):
-        """Energy balance for the feed tray """
+        """Energy balance for the feed tray"""
         return (
-            sum(m.feed[c] * (
-                m.H_L_spec_feed[c] * (1 - m.feed_vap_frac) +
-                m.H_V_spec_feed[c] * m.feed_vap_frac)
-                for c in m.comps) +
             sum(
-                # Heat of liquid from tray above
-                m.L[c, t + 1] * m.H_L[c, t + 1]
-                # heat of liquid to tray below
-                - m.L[c, t] * m.H_L[c, t]
-                # heat of vapor from tray below
-                + m.V[c, t - 1] * m.H_V[c, t - 1]
-                # heat of vapor to tray above
-                - m.V[c, t] * m.H_V[c, t]
-                for c in m.comps)) * 1E-3 == 0
+                m.feed[c]
+                * (
+                    m.H_L_spec_feed[c] * (1 - m.feed_vap_frac)
+                    + m.H_V_spec_feed[c] * m.feed_vap_frac  # Heat of feed liquid
+                )  # Heat of feed vapor
+                for c in m.comps
+            )
+            + sum(
+                m.L[c, t + 1] * m.H_L[c, t + 1]  # Heat of liquid from tray above
+                - m.L[c, t] * m.H_L[c, t]  # Heat of liquid to tray below
+                + m.V[c, t - 1] * m.H_V[c, t - 1]  # Heat of vapor from tray below
+                - m.V[c, t] * m.H_V[c, t]  # Heat of vapor to tray above
+                for c in m.comps
+            )
+        ) * 1e-3 == 0 # Convert the result from [kJ/mol] to [MJ/mol]
```

Comment:
I would split this line to have the comment after 1e-3 and then ==0 in the next line